README.md

```diff
@@ -1,38 +1,75 @@
+ <img alt="npm" src="https://img.shields.io/npm/v/@threlte/core?color=fe4100&labelColor=171d27&logo=npm&logoColor=white"/>
+ <img alt="discord" src="https://img.shields.io/discord/985983540804091964?label=discord&color=fe4100&labelColor=171d27&logo=discord&logoColor=white"/>
+ <img alt="docs" src="https://img.shields.io/website?down_color=red&down_message=offline&label=docs&color=fe4100&labelColor=171d27&up_message=online&url=https%3A%2F%2Fthrelte.xyz&logo=svelte&logoColor=white"/>
+</div>
 
-A three.js component library for svelte.
+<a href="https://threlte.xyz">
+ <img src="./assets/threlte/banner/threlte-banner.png"/>
+</a>
 
-![npm-license](https://img.shields.io/npm/l/@threlte/core?style=flat-square)
-![npm-version](https://img.shields.io/npm/v/@threlte/core?style=flat-square)
-![discord](https://img.shields.io/discord/985983540804091964?label=discord&style=flat-square)
-![website](https://img.shields.io/website?down_color=red&down_message=offline&label=threlte.xyz&style=flat-square&up_message=online&url=https%3A%2F%2Fthrelte.xyz)
+## Create 3D Apps for the Web
 
-Build three.js scenes declaratively with Threlte components that are reactive to the core, readily interactive and can participate in Svelte's ecosystem.
+Threlte is a [Svelte](https://svelte.dev/) library that simplifies creating 3D apps for the web. It provides a **declarative**, **type-safe**, **reactive** and **interactive** API out-of-the-box.
 
-## Official Documentation
+Threlte's core **3D rendering** is powered by [Three.js](https://threejs.org/), but its also provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/). 
+
+Check out our **[documentation](https://threlte.xyz)** and our **[Discord community](https://discord.gg/EqUBCfCaGm)**.
+
+## Quickstart
+
+### Installation
+
+For a quick interactive setup of a fresh Threlte project, run:
+
+```sh
+npm create threlte my-project
+```
+
+Alternatively you can check out the full [installation instructions](https://threlte.xyz/docs/learn/getting-started/installation).
+
+### Tutorial
+
+To get a hang of the basics, we recommend following our [introductory tutorial](https://threlte.xyz/docs/learn/getting-started/your-first-scene).
+
+### Support
+
+Have questions? Feel free to ask in our [Discord support forum](https://discord.com/channels/985983540804091964/1031843197963477002).
+
+## Features
+
+ - [Clean API](https://threlte.xyz/docs/reference/core/t) with a single component transparently exposing *all* Three.js objects through declarative API.
+
+ - [Hooks](https://threlte.xyz/docs/learn/basics/hooks) simplifying access to the underlying Three.js contexts such as animation frames and the rendering engine.
+
+ - [Plugins](https://threlte.xyz/docs/learn/advanced/plugins) making it easy to extend Threlte with custom code and logic.
+
+ - [Events](https://threlte.xyz/docs/learn/basics/events) lets you access state transformations in your scenes.
```

Comment:
This sounds a bit odd, maybe "Events enabling robust and Svelte-native access to state transformations occurring within your scenes." or along these lines?